regulations/static/regulations/css/less/module/drawer-content.less

```diff
@@ -34,20 +34,36 @@ This contains all of the styles specific to the TOC
 
       // to account for the last child padding
       // when there's a nested list
-      li ol li:first-child {
+      li ol li {
         a {
-          padding-top: 0;
+          padding-top: 5px;
+          padding-bottom: 5px;
+        }
+        &:first-child {
+          a {
+            padding-top: 5px;
+          }
+        }
```

Comment:
good catch, it was left over from when that used to be a different value. 

in the process of changing this though I saw some of the rules weren't taking the way   I intended because of the use of descendant combinators over child combinators. going to push something up for that in a sec